Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewController.swift

```diff
@@ -737,9 +741,12 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
     }
   }
 
-  private func presentBlockUserAlert(username: String) {
+  private func presentBlockUserAlert(username: String, userId: Int) {
     let alert = UIAlertController
-      .blockUserAlert(username: username, blockUserHandler: { _ in self.viewModel.inputs.blockUser() })
+      .blockUserAlert(username: username, blockUserHandler: { _ in
+        self.viewModel.inputs.blockUser(id: "\(userId)")
```

Comment:
Should the `viewModel` take an `int` as input instead?

Reply:
Well, the mutation expects a string. I could convert it to a string in the view model. Do you have a strong preference here?